backend/routes/signUpAuth.ts

```diff
@@ -4,37 +4,33 @@ import prisma from '../utils/prismaClient';
 const signUpAuthRouter = express.Router()
 const db = prisma;
 
-signUpAuthRouter.post('/', async function(req, res, next) {
-    console.log(req.body);
-    console.log("hello");
-    const existingUser = await db.user.findUnique({
-        where: {    
-            email: req.body.email
-        }
-    });
-    
-    if (existingUser){
-        res.status(409).send('Existing email. Please use a different email.')
-    }
-    else {
-        const salt = crypto.randomBytes(16).toString("base64");
-        crypto.pbkdf2(req.body.password, salt, 310000, 32, 'sha256', async function(err, hashedPassword) { 
-            if(err) {
-                next(err); 
-            }
-            const user = await db.user.create({
-                data: {
-                    email: req.body.email,
-                    firstName: req.body.firstName,
-                    lastName: req.body.lastName, 
-                    password: hashedPassword.toString("base64"),
-                    salt 
-                },
-            })
-            console.log(user); //redirect to the login page 
-        })
-        res.status(200);
+signUpAuthRouter.post('/', async function (req, res, next) {
+  const existingUser = await db.user.findUnique({
```

Comment:
not blocking but this would be better off abstracted into a method in a `model` file since it is directly interacting with the db and we shouldn't do that directly in the route hander

Reply:
added as a todo!